druid/src/core.rs

```diff
@@ -338,6 +345,14 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                 Event::Timer(*id)
             }
             Event::Command(cmd) => Event::Command(cmd.clone()),
+            Event::TargetedCommand(target, cmd) => {
+                if *target == Target::Widget(self.id) || target.is_window() {
+                    Event::Command(cmd.clone())
+                } else {
+                    // TODO: here's where Bloom filter magic would happen
```

Comment:
what is the mechanism by which children register their ids with parents? I could imagine this potentially being part of `LifeCycle`?

Reply:
Yes, there's got to be a call on `EventCtx` and `UpdateCtx` (contexts in which a widget can dynamically add children) to register the child.